packages/desktop-client/src/components/accounts/Reconcile.jsx

```diff
@@ -59,33 +60,42 @@ export function ReconcilingMessage({
                 marginRight: 3,
               }}
             />
-            All reconciled!
+            <Trans>All reconciled!</Trans>
           </View>
         ) : (
           <View style={{ color: theme.tableText }}>
             <Text style={{ fontStyle: 'italic', textAlign: 'center' }}>
-              Your cleared balance{' '}
-              <strong>{format(cleared, 'financial')}</strong> needs{' '}
-              <strong>
-                {(targetDiff > 0 ? '+' : '') + format(targetDiff, 'financial')}
-              </strong>{' '}
-              to match
-              <br /> your bank’s balance of{' '}
-              <Text style={{ fontWeight: 700 }}>
-                {format(targetBalance, 'financial')}
-              </Text>
+              <Trans>
```

Comment:
This is causing some sort of problem, but there's no error. For me, the reconcile window immediately closes.

Could anyone else confirm please?

Reply:
You can enable debug output in the console by setting `debug: true` in the options in `i18n.ts`.

Reply:
This persists in edge, not a symptom of this PR.